lib/mindwendel/help.ex

```diff
@@ -8,6 +8,8 @@ defmodule Mindwendel.Help do
   alias Mindwendel.Help.Inspiration
 
   def random_inspiration do
-    Repo.one(from t in Inspiration, order_by: fragment("RANDOM()"), limit: 1)
+    if Application.fetch_env!(:mindwendel, :options)[:brainstorming_teasers] == "true" do
```

Comment:
I recommend that `Application.fetch_env!(:mindwendel, :options)[:brainstorming_teasers]` returns a boolean whether this feature is enabled or not. Otherwise, you will always have to check for the value `"true"` in other places as well.